src/Symfony/Bundle/FrameworkBundle/Resources/config/property_access.xml

```diff
@@ -4,10 +4,22 @@
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
 
+    <parameters>
+        <parameter key="property_accessor.cache.prefix" />
+    </parameters>
+
     <services>
         <service id="property_accessor" class="Symfony\Component\PropertyAccess\PropertyAccessor" >
             <argument /> <!-- magicCall, set by the extension -->
             <argument /> <!-- throwExceptionOnInvalidIndex, set by the extension -->
+            <argument /> <!-- cache, set by the extension -->
+        </service>
+
+        <!-- Cache -->
+        <service id="property_accessor.cache.apc" class="Doctrine\Common\Cache\ApcCache" public="false">
```

Comment:
I think that having built-in APCu caches for such metadata makes sense (for PropertyAccess as well as for the Validator and Serializer). AFAIK APCu is one of the fastest Doctrine Cache backend available and to use it only the APCu is needed (it is installed on a large amount of productions servers).

Other backends are less relevant for such metadata. Even when using a cloud computing architecture, it is generally preferable to store this kind of cache (metadata about the code structure) locally on each server. It avoids some network connections (Redis, Memcache...) and can be stored directly in RAM.